packages/@vuepress/markdown/lib/preWrapper.js

```diff
@@ -8,12 +8,15 @@
 //   4. <!--afterend-->
 
 module.exports = md => {
-  const fence = md.renderer.rules.fence
-  md.renderer.rules.fence = (...args) => {
+  const wrap = (wrapped) => (...args) => {
     const [tokens, idx] = args
     const token = tokens[idx]
-    const rawCode = fence(...args)
+    const rawCode = wrapped(...args)
     return `<!--beforebegin--><div class="language-${token.info.trim()} extra-class">`
     + `<!--afterbegin-->${rawCode}<!--beforeend--></div><!--afterend-->`
   }
+  const fence = md.renderer.rules.fence
```

Comment:
Actually, I think you could make a small improvement :

```
const { fence, codeBlock } =  md.renderer.rules
md.renderer.rules.fence = wrap(fence)
md.renderer.rules.code_block = wrap(codeBlock)
```

What do you think @pyaillet ?

Reply:
I need to rename `codeBlock` to `code_block` to do this.
And sadly, the linter doesn't like it:
```
vuepress/packages/@vuepress/markdown/lib/preWrapper.js
  18:18  error  Identifier 'code_block' is not in camel case  camelcase

✖ 1 problem (1 error, 0 warnings)
```

Reply:
Oh sure, I missed that 😉